pkg/streamingpromql/benchmarks/benchmarks.go

```diff
@@ -275,6 +275,12 @@ func TestCases(metricSizes []int) []BenchCase {
 		{
 			Expr: "topk by (le) (5, h_X)",
 		},
+		{
+			Expr: "quantile(0.9, a_X)",
+		},
+		{
+			Expr: "quantile by (l) (0.1, b_X)",
```

Comment:
`l` is the index of each series (eg. `b_100` will have `b_100{l="0"}`, `b_100{l="1"}` etc.), so this will produce X series, one for each input series. 

Perhaps a more interesting benchmark would be grouping to a smaller number of output groups? I'm imagining something like `quantile by (le) (0.1, h_X)`.